app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,13 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
 
-  def show; end
+    posts_list = Post.all.order(created_at: :desc)
+    @posts = posts_list.select { |i| i['id'] == @user[:id] }
+  end
+
+  def show
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:id])
```

Comment:
- [ ] <b>REQUIRED:</b> Why define `@user` here if you weren't going to use it? Kindly remove it.